src/instructlab/model/chat.py

```diff
@@ -195,56 +195,27 @@ def chat(
     # in server mode (`ilab model serve` is running somewhere, or we are talking to another
     # OpenAI compatible endpoint).
     if not is_temp_server_running():
-        # Try to get the model name right if we know we're talking to a local `ilab model serve`.
-        #
-        # If the model from the CLI and the one in the config are the same, use the one from the
-        # server if they are different else let's use what the user provided
-        #
-        # 'model' will always get a value and never be None so it's hard to distinguish whether
-        # the value came from the user input or the default value.
-        # We can only assume that if the value is the same as the default value and the value
-        # from the config is the same as the default value, then the user didn't provide a value
-        # we then compare it with the value from the server to see if it's different
-        if (
-            # We need to get the base name of the model because the model path is a full path and
-            # the once from the config is just the model name
-            os.path.basename(model) == cfg.DEFAULTS.GGUF_MODEL_NAME
-            and os.path.basename(ctx.obj.config.chat.model)
-            == cfg.DEFAULTS.GGUF_MODEL_NAME
-            and api_base == ctx.obj.config.serve.api_base()
-        ):
```

Comment:
What's the reason to stop honoring the `-m` flag here? This looks like a regression.